<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>310</y>
      <width>77</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Read file:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_file_output_name">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>310</y>
      <width>178</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_file_output_ok">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>310</y>
      <width>93</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ok</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>270</y>
      <width>84</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Write file:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_file_input_name">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>270</y>
      <width>171</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>C:\Qt\f.txt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_file_input_ok_2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>270</y>
      <width>93</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ok</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_delete">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>390</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>366</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_delete_all">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>510</y>
      <width>366</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Delete all restaurants</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>371</width>
      <height>75</height>
     </rect>
    </property>
    <property name="title">
     <string>Add</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Restaurant:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_add_2"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_add_ok">
         <property name="text">
          <string>Ok</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_size">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>130</y>
      <width>361</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>count</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>360</y>
      <width>231</width>
      <height>75</height>
     </rect>
    </property>
    <property name="title">
     <string>Erase</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>30</y>
       <width>67</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Restaurant:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="resDel">
     <property name="geometry">
      <rect>
       <x>88</x>
       <y>30</y>
       <width>137</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="addToSet">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>450</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить множество</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>815</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
